central/cloudsources/service/service_impl_postgres_test.go

```diff
@@ -399,6 +408,53 @@ func (s *servicePostgresTestSuite) TestDeleteCloudSource() {
 	s.Assert().ErrorIs(err, errox.NotFound)
 }
 
+func (s *servicePostgresTestSuite) TestCloudSourceTest() {
+	cloudSourceClientMock := cloudSourceClientMocks.NewMockClient(gomock.NewController(s.T()))
+	s.service.(*serviceImpl).clientFactory = func(_ *storage.CloudSource) cloudsources.Client {
+		return cloudSourceClientMock
+	}
+
+	cloudSource := fixtures.GetV1CloudSource()
+
+	// 1. Call test without UpdateCredentials on a non-existent cloud source.
+	_, err := s.service.TestCloudSource(s.writeCtx, &v1.TestCloudSourceRequest{
+		CloudSource: cloudSource,
+	})
+	s.ErrorIs(err, errox.InvalidArgs)
```

Comment:
```suggestion
	s.Assert().ErrorIs(err, errox.InvalidArgs)
```

Reply:
But why? We don't overwrite the assertion function of the suite, and by default the assertion is `assert.New()` and not `require.New`, so this seems unnecessary.

Reply:
Maybe I misunderstood your point here https://github.com/stackrox/stackrox/pull/9542#discussion_r1473656960, but I thought it was about consistency.